app/actions/remote/post.ts

```diff
@@ -26,6 +26,83 @@ type AuthorsRequest = {
     error?: unknown;
 }
 
+export const createPost = async (serverUrl: string, post: Partial<Post>, files: FileInfo[] = []): Promise<{data?: boolean; error?: any}> => {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
+        return {error: `${serverUrl} database not found`};
+    }
+
+    const currentUserId = queryCurrentUserId(operator.database);
+    const timestamp = Date.now();
+    const pendingPostId = post.pending_post_id || `${currentUserId}:${timestamp}`;
+
+    // Check if is sending this post id
+
+    let newPost = {
+        ...post,
+        id: '',
+        pending_post_id: pendingPostId,
+        create_at: timestamp,
+        update_at: timestamp,
+        ownPost: true,
```

Comment:
I don't think we need this at the server or locally

Reply:
I guess you mean the ownPost attribute. Seems like it is not being used in many places on V1, and where it is used, usually is calculated from the current and post user id. Will remove.